alerting/pagerduty/integrations-pagerduty.tf

```diff
@@ -0,0 +1,5 @@
+resource "signalfx_pagerduty_integration" "sfx_integration" {
+  name = "PagerdutyIntegration${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
+	enabled = var.enabled
```

Comment:
tf fmt?

Reply:
already done but not pushed sorry, now everything is ready for review